.gitignore

```diff
@@ -12,7 +12,6 @@ server/dist/**
 **/node_modules/**
 
 # Local env.
-.vscode
```

Comment:
we could perhaps limit this only allow launch.json only (there are other vscode files which we don't want to check in yet..)

Reply:
only adding launch.json now